src/lib/evil/evil_unistd.h

```diff
@@ -62,8 +62,9 @@ EVIL_API int ftruncate(int fd, off_t size);
 EVIL_API double evil_time_get(void);
 
 #ifdef _MSC_VER
-# include <stdint.h>
```

Comment:
we don't need stdint anymore?

Reply:
> * stdint.h include removed because it was only needed because of uint64_t

Reply:
Why not using `uint64_t` again (instead of `unsigned long`)?

Reply:
> > * stdint.h include removed because it was only needed because of uint64_t

oh, please add this to the PR description as well.

Reply:
> Why not using `uint64_t` again (instead of `unsigned long`)?

I don't understand the point of including another header to define a type that we know that is a fixed-width integral. Actually shouldn't be `unsigned long long`?

Reply:
Mostly because `uint64_t` is the way to go in these cases, since:
- It is platform independent (this is the main point);
- It is what we should use if we want an integral data type with specific length (which enforces the previous point).

But seeing [mingw's sys/types.h](https://sourceforge.net/p/mingw-w64/mingw-w64/ci/master/tree/mingw-w64-headers/crt/sys/types.h#l84), it defines `useconds_t` to `unsigned int`. Maybe that should be its final type? What's POSIX's one?

Reply:
> The type useconds_t shall be an unsigned integer type capable of storing values at least in the range [0, 1000000]. The type suseconds_t shall be a signed integer type capable of storing values at least in the range [-1, 1000000].

ref: https://pubs.opengroup.org/onlinepubs/007904975/basedefs/sys/types.h.html

Reply:
Hm, in that case, `unsigned int` (which is guaranteed by C and C++ specifications to have >= 4 byte size) is more than enough, since 1000000 fits in 20 bits.

Reply:
done